docs/development/docrules.rst

```diff
@@ -1,569 +1,8 @@
-.. doctest-skip-all
-.. _doc-rules:
+:orphan:
```

Comment:
I'm not a big fan of keeping  orphan pages. Why not delete it all?

Reply:
In case any other projects were linking to it, to avoid breaking links for now.

Reply:
I am pretty sure sunpy is linking to this.

Reply:
Then they better update their links? 🤷‍♀

Reply:
I have merged this now, but my comment still stands. I don't think we should keep orphaned pages around, downstream should keep track of their links and update as needed.